.github/workflows/jerigon-native.yml

```diff
@@ -58,41 +57,37 @@ jobs:
 
       #It is much easier to use cast tool in scripts so install foundry
       - name: Install Foundry
-        uses: foundry-rs/foundry-toolchain@v1          
+        uses: foundry-rs/foundry-toolchain@v1
 
       - name: Run cancun test network
         run: |
           docker pull ghcr.io/0xpolygonzero/erigon:feat-zero
-          kurtosis run --enclave cancun-testnet github.com/ethpandaops/ethereum-package@4.0.0 --args-file jerigon-test-network/network_params.yml  
+          kurtosis run --enclave cancun-testnet github.com/ethpandaops/ethereum-package@4.0.0 --args-file jerigon-test-network/network_params.yml
 
       - name: Generate blocks with transactions
         run: |
-            ETH_RPC_URL="$(kurtosis port print cancun-testnet el-2-erigon-lighthouse ws-rpc)"
-            cast rpc eth_blockNumber --rpc-url $ETH_RPC_URL
-            cd jerigon-test-network && set -a && source .env && set +a
-            bash ./tests/generate_transactions.sh
- 
+          ETH_RPC_URL="$(kurtosis port print cancun-testnet el-2-erigon-lighthouse ws-rpc)"
+          cast rpc eth_blockNumber --rpc-url $ETH_RPC_URL
+          cd jerigon-test-network && set -a && source .env && set +a
+          bash ./tests/generate_transactions.sh
+
       - name: Run prove blocks with native tracer in test_only mode
         run: |
           ETH_RPC_URL="$(kurtosis port print cancun-testnet el-2-erigon-lighthouse ws-rpc)"
           cd zero_bin/tools
           ulimit -n 8192
-          OUTPUT_TO_TERMINAL=true ./prove_rpc.sh 0x5 0xf $ETH_RPC_URL native true 3000 100 test_only
+          OUTPUT_TO_TERMINAL=true ./prove_rpc.sh 0x1 0xf $ETH_RPC_URL native true 3000 100 test_only
           echo "Proving blocks in test_only mode finished"
 
-
       - name: Run prove blocks with native tracer in real mode
         run: |
           ETH_RPC_URL="$(kurtosis port print cancun-testnet el-2-erigon-lighthouse ws-rpc)"
           cd zero_bin/tools
           rm -rf proofs/* circuits/* ./proofs.json test.out verify.out leader.out
-          OUTPUT_TO_TERMINAL=true RUN_VERIFICATION=true ./prove_rpc.sh 0x5 0x7 $ETH_RPC_URL native true 3000 100
+          OUTPUT_TO_TERMINAL=true RUN_VERIFICATION=true ./prove_rpc.sh 0x1 0xf $ETH_RPC_URL native true 3000 100
```

Comment:
We were intentional proving only a few blocks in real mode because test was too long otherwise. You can return value to `0x4 0x7`

Reply:
I think we should start atleast with 0x1 so that 0x1, 0x4 both cases are covered

Reply:
That is covered with `test_only` test

Reply:
ok makes sense